### PR TITLE
FateSummaryIT Jenkins failure fix attempt

### PR DESCRIPTION
**Pull request by kevinrr888**
part of https://github.com/apache/accumulo/issues/4614

This is an attempt to fix the Jenkins build failure for this test. Locally, I have not been able to reproduce this test timeout (only takes about 1 minute locally and passes). So, I could not test that these changes fix the problem. If anyone knows how I can better test these changes, that would be great.

Since it was timing out at the deletion of the table, I thought this might be caused by creating, using, and deleting a table with the same name twice. I can't think of any other reason this might be timing out here...

### CODE REVIEW

**Comment by dlmarion** [comment]
I don't know if this will fix it, as the delete is still there inside the loop, so it could still cause the test to timeout.

Also, looking at this code:
https://github.com/apache/accumulo/blob/d6eb1df8ee3155dfbf1fe03548433d6a61566f8f/test/src/main/java/org/apache/accumulo/test/FateSummaryIT.java#L197-L211

I think your assumption is off by a factor of 10. You are inserting 10 rows, but with 10 columns each, so your compaction will iterate over 100 key/value pairs.

**Comment by kevinrr888** [comment]
Update: I've spoken with Ed, Dom, and Arbaaz about this. Ed has the same failure running locally: the timeout on https://github.com/apache/accumulo/blob/2.1/test/src/main/java/org/apache/accumulo/test/FateSummaryIT.java#L235
Dom, Arbaaz, and I have not been able to reproduce this locally (passes for us - the delete table takes about 10-13 seconds). Dom and I are thinking it is a hardware difference in machines (Dom, Arbaaz, and I have the same machines). Perhaps my table is too large making the delete take too long on machines with less resources. Working on new fix

**Comment by EdColeman** [comment]
It looks like the table delete time is proportional to the number of transactions.  Using 125 transactions the delete completed in 3 seconds.  With 250 it took 30 seconds.  So, either the delete is waiting on transactions to complete before it can get a lock - or if it is calling cancel, then that takes longer with more transactions.

I tried a version that waited for 0 transactions before issuing the delete and it would need to wait for about the same time as the delete is taking.  

Another observation from that test, the admin command seems fairly heavy weight.  If I slept 500 ms between transaction count checks, the admin command would show up in top with cpu usage pegged.

**Comment by kevinrr888** [comment]
I decreased the number of txns from 500 to 250, for me, this decreased the time to delete the table from 10-15 seconds down to 2-3 seconds. Hoping this speedup is enough for Jenkins to run this test successfully. For Ed, 500 txns caused a timeout, and 250 did not, so hoping Jenkins is similar. Probably shouldn't go much lower than 250 txns since less may not give enough attempts at running the print cmd before all the transactions complete. I have only ever seen it run at most 2 iterations of the do-while loop and I have run it quite a bit, but it is possible for more iterations. I tried running the test with 100 txns, and this would occasionally give me a failure when that 2nd iteration occurred (they would all be completed by then). Honestly, 250 txns would probably fail if 3 or more attempts had to occur but I think this is rare enough where it is okay. For example, on a run where 250 txns are created: 184 were still in progress at the first print attempt (which did not see a txn complete mid print), then 77 were in progress at the second attempt which succeeded. If the second attempt didn't see a txn complete mid print, all of the txns would have probably completed and the test would have failed. Out of my last 10 runs, two iterations only occurred once so a third attempt is probably around a 1% chance. This is originally why I went with 500 transactions. I don't know if this chance of failure would be considered too flaky to have for a test.

TLDR: Using 250 transactions instead of 500 has a higher chance of failure probably around a 1% chance. Is this too flaky for a test?

**Comment by keith-turner** [comment]
> TLDR: Using 250 transactions instead of 500 has a higher chance of failure probably around a 1% chance. Is this too flaky for a test?

That is probably fine.  The test are run up to 5 times on failure, so if its 1% chance of failure per run then its very unlikely it will fail 5 times in a row.

**Review by keith-turner — APPROVED** [review]

**Comment by keith-turner** [comment]
Merged these changes to main and elasticity.  When merging to elasticity I confirmed this was the only outstanding commit and did `git merge -sours main` to create f2c8c833e1d1a024ac74ad09884ceb666a2a71d6 in the elasticity branch.

**Comment by kevinrr888** [comment]
> > TLDR: Using 250 transactions instead of 500 has a higher chance of failure probably around a 1% chance. Is this too flaky for a test?
> 
> That is probably fine. The test are run up to 5 times on failure, so if its 1% chance of failure per run then its very unlikely it will fail 5 times in a row.

Oh got it didn't know they were retried
